tests/tests.go

```diff
@@ -209,6 +210,26 @@ func initKubemacpoolParams(rangeStart, rangeEnd string) error {
 	return nil
 }
 
+func getVmFailCleanupWaitTime() time.Duration {
+	managerDeployment, err := testClient.KubeClient.AppsV1().Deployments(managerNamespace).Get(names.MANAGER_DEPLOYMENT, metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred(), "Should succeed getting Deployment object")
+	Expect(len(managerDeployment.Spec.Template.Spec.Containers) != 0).To(BeTrue())
```

Comment:
Please add Expect message "Container count should be greater then zero / > 0 .."